tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -172,9 +173,9 @@ protected boolean init(String[] args) throws Exception {
     amAddress = cliParser.getOptionValue("am_address", "");
     taskCommand = cliParser.getOptionValue("task_command", "exit 0");
     timeOut = tonyConf.getInt(TonyConfigurationKeys.WORKER_TIMEOUT,
-        TonyConfigurationKeys.DEFAULT_WORKER_TIMEOUT);
+                              TonyConfigurationKeys.DEFAULT_WORKER_TIMEOUT);
```

Comment:
Can we keep the indentation like this? The 4 spaces thing for continuing long lines is our convention (here and everywhere else)

Reply:
ok.. I guess I need to update my Intellij preference. I did a copy paste for this whole file